extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -88,17 +110,28 @@ public OpenCensusProtobufInputRowParser withParseSpec(ParseSpec parseSpec)
       throw new ParseException(e, "Protobuf message could not be parsed");
     }
 
+    // Fetch metric name.
+    String metricName = this.hasMetricDimension ? this.metricDimension : metric.getMetricDescriptor().getName();
```

Comment:
I think there was a misunderstanding here, we are not trying to change the name of the metric itself. The point of having `metricDimension` was to change the value of the *dimension*, i.e. the `NAME` constant defined here https://github.com/confluentinc/druid/pull/14/files#diff-13a54faa5722482b59f3f83c9fdfd4deR52

Reply:
Done.